app/code/Magento/Catalog/Ui/Component/Listing/Columns/StockStatus.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Catalog\Ui\Component\Listing\Columns;
+
+use Magento\Framework\View\Element\UiComponentFactory;
+use Magento\Framework\View\Element\UiComponent\ContextInterface;
+
+class StockStatus extends \Magento\Ui\Component\Listing\Columns\Column
+{
+    const NAME = 'quantity_and_stock_status';
```

Comment:
Please add a phpdoc. Constants must have short description according to our standatds: https://devdocs.magento.com/guides/v2.2/coding-standards/docblock-standard-general.html#constants